docker/loki/docker-compose.loki.yml

```
@@ -0,0 +1,26 @@
+      - /Users/sangmin8817/Desktop/haksa/logs:/app/logs
+      - ./promtail-local.yaml:/etc/promtail/config.yaml
+    command: -config.file=/etc/promtail/config.yaml
+
+  grafana:
+    image: grafana/grafana:10.2.3
+    container_name: grafana
+    ports:
+      - "13000:3000"
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=admin
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)